apps/condo/domains/meter/tasks/deleteDeletedMeterMeterReadings.js

```diff
@@ -0,0 +1,26 @@
+const { getSchemaCtx, find } = require('@condo/keystone/schema')
+const { createTask } = require('@condo/keystone/tasks')
+const { MeterReading } = require('../utils/serverSchema')
+
+/**
+ * Soft delete meter readings after soft delete meter
+ */
+async function deleteDeletedMeterMeterReadings (deletedMeter, deletedMeterAt) {
```

Comment:
Could you please change name to more humanized, like 'deleteReadingsOfDeletedMeter'?